lib/src/main/java/com/orange/ods/compose/component/button/OdsFloatingActionButton.kt

```diff
@@ -44,28 +50,26 @@ private val MiniFabSize = 40.dp
  *
  * @see androidx.compose.material.FloatingActionButton for further information.
  *
+ * @param icon The FAB icon.
  * @param onClick callback invoked when this FAB is clicked
  * @param modifier [Modifier] to be applied to this FAB.
  * @param mini If `true`, the size of the FAB will be 40dp, otherwise the default size will be used.
- * @param icon [Painter] of the FAB icon.
- * @param iconContentDescription [icon] content description.
  */
 @Composable
 @OdsComposable
 fun OdsFloatingActionButton(
+    icon: OdsFloatingActionButtonIcon,
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
     mini: Boolean = false,
-    icon: Painter,
-    iconContentDescription: String
 ) {
     FloatingActionButton(
         onClick = onClick,
         modifier = modifier.let { if (mini) it.size(MiniFabSize) else it },
         backgroundColor = OdsTheme.colors.component.floatingActionButton.background,
         contentColor = OdsTheme.colors.component.floatingActionButton.content
     ) {
-        Icon(painter = icon, contentDescription = iconContentDescription)
+        icon.Content(modifier = Modifier.size(FabIconSize))
```

Comment:
Size could be applied in `Content` directly.

Same comment for `ExtendedFloatingActionButton`.